### PR TITLE
fix: Prevent copying workflow when copying outside of canvas

### PR DESCRIPTION
**Pull request by mutdmour**
## Summary
- Prevent copying workflow when focused in Assistant 

## Review / Merge checklist

- [X] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [X] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/AskAssistant/AskAssistantChat.vue [hunk 1]
```diff
@@ -83,6 +83,8 @@ function onClose() {
 				:style="{ width: `${assistantStore.chatWidth}px` }"
 				:class="$style.wrapper"
 				data-test-id="ask-assistant-chat"
+				tabindex="0"
+				@keydown.stop
```
Comment (mutdmour) [inline packages/editor-ui/src/components/AskAssistant/AskAssistantChat.vue:87]: this is the fix for the issue.

**Review by MiloradFilipovic — COMMENTED** [review]
Works as expected for chat but I don't think it addresses the issue in the ticket. The original report doesn't mention the chat (_Add any node and copy some value from it_). Should we create a new ticket for this or look more into it?

**Comment by MiloradFilipovic** [comment]
> @MiloradFilipovic if you look at Linear comments in ticket, you see that we cancelled because no one (tomi/oleg/nick) can reproduce this anymore. then we reopened it just for the assistant. sorry for the confusion.

Ah, didn't check the timeline. Ok, looks good then

**Review by MiloradFilipovic — APPROVED** [review]
Works as expected

**Comment by janober** [comment]
Got released with [`n8n@1.59.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.59.1)

**Pull request merged**